ignite/engine/engine.py

```diff
@@ -37,6 +37,9 @@ class Engine(object):
     Args:
         process_function (Callable): A function receiving a handle to the engine and the current batch
             in each iteration, and returns data to be stored in the engine's state
+        additional_events (Enum, optional): Additional events that can be fired.
```

Comment:
Just wonder whether it makes sense to have a list of custom events ? 
For example, I write another custom Tbptt trainer and want to use your Tbptt_Events plus another CustomEvent...
Maybe a solution will be to inherit of Tbptt_Events ...